src/Const/README.md

```diff
@@ -0,0 +1,217 @@
+// updateField :: a -> Field
+const updateField =
+  value => Pair(Const(value), value)
+
+// updateField :: Field -> Field
+const resetField =
+  extend(compose(valueOf, fst))
+
+const changed =
+  updateField('Joey')
+    .map(toLower)
+    .chain(updateField)
+//=> Pair( Const "Joey", "joey" )
+
+resetField(changed)
+//=> Pair( Const "Joey", "Joey" )
+
+```
+<article id="topic-implements">
+
+## Implements
+`Functor`, `Apply`, `Chain`
+
+</article>
+
+<article id="topic-instance">
+
+## Instance Methods
+
+#### equals
+
+```haskell
+Const c a ~> b -> Boolean
+```
+
+Used to compare the underlying values of (2) `Const` instances for equality by
+value, `equals` takes any given argument and returns `true` if the passed
+arguments is a `Const` with an underlying `left` value equal to the underlying value
+of the `Const` the method is being called on. If the passed argument is not
+a `Const` or the underlying values are not equal, `equals` will return `false`.
+
+```javascript
+import { Const } from 'crocks'
+
+const two = Const(2)
+const four = Const(4)
+
+two.equals(four)
+//=> false
+
+two.equals(Const(2))
+//=> True
+```
+
+#### concat
+
+```haskell
+Const c a ~> Const c a -> Const c a
+```
+
+`concat` is used to combine (2) `Semigroup`s of the same type under an operation
+specified by the `Semigroup`. In the case of `Const`, it will return a new
+`Const` instance with the original value.
+
+```javascript
+import { Const, concat, map } from 'crocks'
+
+const account1 = { firstName: 'John', lastName: 'Doe', achievments: [ '112', '232', '154' ] }
+const account2 = { firstName: 'Jon', lastName: 'Doe', achievments: [ '767', '989' ] }
+
+const constMerge = xs =>
+  map(Const, xs)
+    .reduce(concat)
+    .valueOf()
+
+const reduceAccounts = (acc, cur) => ({
+  firstName: constMerge([ cur.firstName, acc.firstName ]),
+  lastName: constMerge([ cur.lastName, acc.lastName ]),
+  achievments: concat(cur.achievments, acc.achievments)
+})
+
+const mergeAccounts = accounts =>
+  accounts
+    .reduce(reduceAccounts)
+
+mergeAccounts([ account1, account2 ])
+```
+
+#### map
+
+```haskell
+Const c a ~> (a -> b) -> Const c b
+```
+
+In a `Product` type the left side is always fixed and the right side is mapped.
+Due to the unique property of `Const` any function that is passed in to `map`
+will be validated but it will not be evaluated. `map` will return a new `const`
+with the same left value.
+
+```javascript
+import { Const } from 'crocks'
+
+Const('initial')
+  .map(x => x.fakeProperty)
+//=> Const 'initial'
+
+```
+
+#### ap
+
+```haskell
+Const c (a -> b) ~> Const c a -> Const c b
+```
+
+Short for apply, `ap` is normally used to apply a `Const` instance containing a 
+value to another `Const` instance that contains a function, resulting in new 
+`Const` instance with the result. However, due to the unique nature of `Const`
+the function will remain the active value in the `Const`.
+
+```javascript
+import { Const } from 'crocks'
```

Comment:
need to fix the imports to point at `crocks/Const`
(so technically this *will* work for things like Rollup and Babel. but when we move to `esm` we will have to change all of these to things like `crocks/esm/Const/index.mjs`, but for now we will keep how we have been doing it)